xija/tests/test_models.py

```diff
@@ -137,7 +137,7 @@ def test_minusz():
     mdl.make()
     mdl.calc()
 
-    regrfile = 'minusz.npz'
+    regrfile = abs_path('minusz.npz')
```

Comment:
Yes.  The file is bundled and is required to exist.  The existing code makes it easy to regenerate the file if needed.  In the unlikely event of some strange process failure where it gets out of the distribution then we will get the same test fail again.